src/com/aldinrizvo/qamp/oophomework5/ATM.java

```diff
@@ -0,0 +1,71 @@
+package com.aldinrizvo.qamp.oophomework5;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ATM {
+    private final List<Account> accountList;
+
+    public ATM() {
+        this.accountList = new ArrayList<>();
+    }
+
+    public void addAccount(final Account account) {
+        if (!accountExists(account.getSerialNumber())) {
```

Comment:
This is totally correct and in the spirit of what we have learned. The comment is directed if you are interested in more options.

Maybe you want to check out `Set`. A set is a type of collection that holds only unique values. `Set.add()` also returns a `boolean` depending if the added values exists or not.

For this to work you would need to properly implement, `hashCode()` and `equals()` for `Account`s.

Maybe if you find the time you can fiddle around a bit.